eng/SendToHelix.proj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+  <PropertyGroup>
+    <Language>msbuild</Language>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <HelixAccessToken Condition="'$(_isOfficialBuild)' == 'true'">$(_helixAccessToken)</HelixAccessToken>
+    <WaitForWorkItemCompletion Condition="'$(_isOfficialBuild)' == 'false'">true</WaitForWorkItemCompletion>
+    <Creator Condition="'$(_isOfficialBuild)' == 'false'">dotnet-wcf</Creator>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="!$(HelixTargetQueue.StartsWith('Windows'))">
+    <HelixPreCommands>"chmod a+x $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh;sudo -E -n $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"</HelixPreCommands>
```

Comment:
Likely you'll want to prepend these with `$(HelixPreCommands)` to avoid overwriting command that the SDK creates itself. It shouldn't happen, but better safe than sorry.
```suggestion
    <HelixPreCommands>"chmod a+x $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh;sudo -E -n $HELIX_CORRELATION_PAYLOAD/InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file /tmp/wcfrootca.crt"</HelixPreCommands>
```
Also, I do think you should consider sending the cert up as a second correlation payload.

Reply:
The certificate is actually on our server, it gets pulled down from there and installed by this script.

Reply:
ah perfect! That will be fine then